huaweicloud/services/ecs/resource_huaweicloud_compute_interface_attach.go

```diff
@@ -43,13 +47,23 @@ func ResourceComputeInterfaceAttach() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
-
 			"instance_id": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
 
+			"ipv6_bandwidth_id": {
```

Comment:
suggest to add `ipv6_bandwidth_id` and `ipv6_enable` as the last two optional parameters.